CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+---
+explains: How to contribute to this City of Amsterdam Open Source project
+layout: guides
+---
+
+# Contributing
+
+We – the maintainers of this project and [the City of Amsterdam](https://www.amsterdam.nl) – value your input, enjoy feedback and welcome improvements to our Open Source projects.
+
+We love issues and pull requests from everyone.
+
+## Problems, suggestions and questions
+
+You don't need to change any of our code or documentation to be a contributor. Many contributors add to our software by reporting problems, suggesting changes and asking simple and difficult questions. To do this, you can create a [GitHub Issue](https://help.github.com/articles/creating-an-issue/) for this project.
+
+## Documentation and code
+
+If you want to add to the documentation or code of one of our projects you should make a Pull Request. If you never used GitHub, get up to speed with [Understanding the GitHub Flow](https://guides.github.com/introduction/flow/).
+
+### 1. Make your changes
+
+Add your changes in commits [with a message that explains them](https://github.com/alphagov/styleguides/blob/master/git.md#commit-messages). Document choices or decisions you make in the commit message, this will enable everyone to be informed of your choices in the future.
+
+If you are adding code, make sure it adheres to the project's code and documentation style guide before you submit your pull request. You can read more about how to document and style your code in the project's [`README`](README.md).
+
+### 2. Pull Request
```

Comment:
I propose to add a sentence about tests. Write tests that show the behaviour of your newly added or changed code

Reply:
Fixed!

Reply:
Does this solve the problem @Vegter

Reply:
I think so. Although I still feel uncertain about having projects without tests. As you mentioned, this project should be an example of a project without any automated tests. But should it not include a spell check for instance? What is a project without inherent automated quality assurance?